src/shared/mount-util.c

```diff
@@ -453,6 +453,21 @@ int bind_remount_one_with_mountinfo(
         return 0;
 }
 
+int bind_remount_one(const char *path, unsigned long new_flags, unsigned long flags_mask) {
+        _cleanup_fclose_ FILE *proc_self_mountinfo = NULL;
+        int r;
+
+        proc_self_mountinfo = fopen("/proc/self/mountinfo", "re");
+        if (!proc_self_mountinfo)
+                return log_debug_errno(errno, "Failed to open /proc/self/mountinfo: %m");
+
+        r = bind_remount_one_with_mountinfo(path, new_flags, flags_mask, proc_self_mountinfo);
+        if (r < 0)
+                return log_debug_errno(r, "Failed to remount '%s': %m", path);
```

Comment:
i'd drop this debug log message.

bind_remount_one_with_mountinfo() already logs about many errors at debug level anyway, and I think we should rather add more there, to ensure all errors are covered, then adding a redundant log messag here.

Reply:
Done, returning the result of `bind_remount_one_with_mountinfo` directly.